### PR TITLE
Refactor travis conftig to test all supported versions

### PR DESCRIPTION
**Pull request by codingjoe**
I personally think tox is great, but on travis it comes
with some down sides.
If I remembered correctly you had a certain development
flow that's why I didn't refactor the tox file much.
I just wanted travis to reflect all versions that are to
be supported.

I'm happy that I did that, because the 3.5 build just
failed.

Since Travis-CI is the last line of deffence, I think
it is worth the effort here.
Let me know if I missed any version.

I tried to save you double trouble by putting everything
in a requirements-dev.txt file. I could also update the
tox file to use it, but as I said before, I didn't want
to break your development workflow. I just wanted tests
for Python 3.5.

### CODE REVIEW

**Comment by codingjoe** [comment]
@kmmbvnr if you have any Idea on how to fix the bug in python3.5. Please let me know and I can add a second commit that fixes it.

**Comment by kmmbvnr** [comment]
I didn't use travis for anything beyond simple "Build Succeed" badge for the github.

I found highly incontinent to have a build tool w/o ssh access to fight again build only errors.. Right now celery integration tests works for me. To get the actual celery output, and see why they failed, we should run it with `-v3` flag. For travis that mean - do several dumb commits to solve the travis only issue.

```
$ REDIS_CACHE_URL=redis://127.0.0.1:6379/1 DATABASE_URL=postgres://postgres:postgres@localhost/viewflow TOXENV=py35-dj18 tpm test tests.test_contrib_celery.Test
py35-dj18 runtests: commands[0] | python manage.py test tests.test_contrib_celery.Test
Creating test database for alias 'default'...
..
----------------------------------------------------------------------
Ran 2 tests in 5.015s

OK
```

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#
+# This file is autogenerated by pip-compile
+# Make changes in requirements-dev.in, then run this to update:
+#
+#    pip-compile requirements-dev.in
+#
+-e .
+-e git+https://github.com/viewflow/django-material.git@master#egg=django-material
+amqp==1.4.9               # via kombu
```
Comment (kmmbvnr) [inline requirements-dev.txt]: I'm pretty happy with single tox files for that

File: tox.ini [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{32,33}-{dj16,dj18}
-    py{34,35}-{dj18,dj19}
+    py{34,35}-{dj18,dj19,djma}
```
Comment (kmmbvnr) [inline tox.ini]: No

**Comment by kmmbvnr** [comment]
Please use `tox` for tests.

**Comment by kmmbvnr** [comment]
And one more remainder, please open an issue on a github b/f starting to work on changes.

https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/

**Comment by codingjoe** [comment]
@kmmbvnr good point, thanks for the post. I opened up an issue. We really need to tackle this. I can't stress this enough. Is an absolute must have for us.

**Pull request closed without merging**